src/commands/deploy.ts

```diff
@@ -219,7 +220,16 @@ export default class Deploy extends Command {
     const config_payload = classToPlain(await EnvironmentConfigBuilder.buildFromPath(env_config_path));
 
     cli.action.start(chalk.blue('Creating deployment'));
-    const { data: deployment } = await this.app.api.post(`/environments/${all_answers.environment_id}/deploy`, { config: config_payload });
+
+    let deployment;
+    try {
+      deployment = (await this.app.api.post(`/environments/${all_answers.environment_id}/deploy`, { config: config_payload })).data;
```

Comment:
Can we make our api's response flatten the errors so we don't need to have logic in the cli/app?